extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
```

Comment:
Ah ok, so you either have an `<amp-state>` with just the src attribute and no children, or an `<amp-state>` with JSON child with optional `src` and `credential` attributes? Done.